build.sbt

```diff
@@ -1,11 +1,12 @@
 val disciplineMUnitVersion = "1.0.9"
-val lucumaCoreVersion      = "0.76.0"
+val lucumaCoreVersion      = "0.80.0"
+val lucumaODBSchema        = "0.3.0"
```

Comment:
Are we getting decoders from the odb now?

Reply:
Yes